docsite/docs.css

```diff
@@ -48,10 +48,6 @@ aside>h1::after{
     bottom: 5px;
     display: inline-block;
     
-}
-aside > ul > li > a {
-    
-
 }
 aside > ul > li > a:hover{
```

Comment:
can give `a` tag class like `aside-link` or `nav-link` instead of using combinators

Reply:
Done! Thanks for reviewing :)